Library/ViewModels/FacebookConfirmationViewModel.swift

```diff
@@ -113,25 +113,25 @@ FacebookConfirmationViewModelErrors {
       self.viewDidLoadProperty.signal.map { AppEnvironment.current.countryCode == "US" }
     ])
 
-    let signupEvent = combineLatest(self.facebookTokenProperty.signal, self.sendNewsletters)
+    let signupEvent = Signal.combineLatest(self.facebookTokenProperty.signal, self.sendNewsletters)
       .takeWhen(self.createAccountButtonProperty.signal)
       .switchMap { token, newsletter in
         AppEnvironment.current.apiService.signup(facebookAccessToken: token, sendNewsletters: newsletter)
           .on(
-            started: {
+            starting: {
               isLoading.value = true
             },
             terminated: {
               isLoading.value = false
           })
-          .delay(AppEnvironment.current.apiDelayInterval, onScheduler: AppEnvironment.current.scheduler)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
           .materialize()
     }
 
     self.logIntoEnvironment = signupEvent.values()
 
     self.postNotification = self.environmentLoggedInProperty.signal
-      .mapConst(NSNotification(name: CurrentUserNotifications.sessionStarted, object: nil))
+      .mapConst(.init(name: .init(rawValue: CurrentUserNotifications.sessionStarted), object: nil))
```

Comment:
yeah i agree. i think i did this to get under the 110 line count limit, but will find a way to make it work.